index.html

```diff
@@ -0,0 +1,194 @@
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+              </ul>
+              <div class="nav-menu-link">
+                <a href="#"> TS Campaign</a>
+              </div>
+            </div>
+          </div>
+        </nav>
+      </div>
+
+      <div class="showcase index">
+        <blockquote>&ldquo; Hello! Fashion Peeps &rdquo;</blockquote>
+        <h1>CREATIVE STITCHES GLOBAL CONFERENCE 2021</h1>
+
+        <div class="para-border">
+          <p>
+            A great celebration for the fashion like minds, where we share tip
+            and open your heart to new insight of
+            <span class="highlight">fashion </span>
+          </p>
+        </div>
+        <div class="venue">
+          <h2>2021.10.15(THU) ~ 15(SAT)</h2>
+          <p>@ National park Lagos Nigeria.</p>
+        </div>
+      </div>
+    </header>
+
+    <main>
+      <section class="main">
+        <div class="program">
+          <div class="main-heading">
+            <h1>Main Program</h1>
+            <div class="sm-line"></div>
+          </div>
+          <section class="program-list">
+            <article class="li">
+              <img src="./images/lecture.png" alt="fashionicons" />
+              <h4>Lecture</h4>
+              <p>
+                We teach you, what it takes to know about fashion and be
+                fashionable. You can proudly pick colors and trend
+              </p>
+            </article>
+            <article class="li">
+              <img src="./images/face.png" alt="fashionicons" />
+              <h4>Men Fashion Exhibition</h4>
+              <p>
+                Men fashin is its own unique style, Talks on cloths that brings
+                the calm and bold you.
+              </p>
+            </article>
+            <article class="li">
+              <img src="./images/lady.png" alt="fashionicons" />
+              <h4>Women Fashion Exhibition</h4>
+              <p>
+                Learn to dress and look elegant as a woman, fly your colors
+                correctly
+              </p>
+            </article>
+            <article class="li">
+              <img src="./images/ropes.png" alt="fashionicons" />
+              <h4>The Fashion Way</h4>
+              <p>Discussions on the trend and hw to make it better</p>
+            </article>
+            <article class="li">
+              <img src="./images/tie.png" alt="fashionicons" />
+              <h4>Tie The knot</h4>
+              <p>
+                Rap up with communication and networking, get to meet leading
+                fashion tycoon around the world
+              </p>
+            </article>
+          </section>
+        </div>
+        <div class="summit">
+          <a href="#" class="hide-on-desktop">
+            Join Touch-Stiches Summit 2021</a
+          >
+          <a href="#" class="hide-on-mobile">See the whole program</a>
+        </div>
+      </section>
```

Comment:
### OPTIONAL 
_I make it as a optional but these are requirements that missing. So please fix them for make it better project._

- [ ] You should stick to the design. You should change font family, background-image, space between elements. You should do these changes on mobile version also.

### ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/146976273-716fca6e-a0fa-4f4c-a787-75ec4847353c.png)

___
### YOUR DESIGN
![image](https://user-images.githubusercontent.com/38635158/146976204-f77fd696-6846-4898-96c5-3d8648188f95.png)

### ORIGINAL MOBILE
![image](https://user-images.githubusercontent.com/38635158/146978061-b277f63b-8c7a-4aea-bda9-6bd65e5a7b7a.png)
___
### YOUR DESIGN MOBILE
![image](https://user-images.githubusercontent.com/38635158/146978113-f4a12310-1646-4ac8-9bb3-2ebd291cf837.png)